test/roots/pillar/sudoers.sls

```diff
@@ -0,0 +1,51 @@
+sudoers:
+  users:
+    root:
+      - 'ALL=(root) NOPASSWD: ALL'
+    vagrant:
+      - 'ALL=(ALL) ALL'
+  groups:
```

Comment:
I added vagrant as this file only gets used inside of vagrant. When it is time to write tests, this file can be modified further.